Assets/MixedRealityToolkit-SDK/Inspectors/MixedRealityToolkit.SDK.Inspectors.asmdef

```diff
@@ -3,7 +3,7 @@
     "references": [
         "Microsoft.MixedReality.Toolkit",
         "Microsoft.MixedReality.Toolkit.InputSystem",
-        "Microsoft.MixedReality.Toolkit.SDK"
+        "Microsoft.MixedReality.Toolkit.SDK.Input"
```

Comment:
Inspectors are for the SDK level. 

We will need another custom assembly that targets the sdk layer or rename the SDK.Inspectors to only reference SDK.Input.

Reply:
Wouldnt inspectors reference all SDK assemblies?